node_lambnik/README.md

```diff
@@ -0,0 +1,10 @@
+# tilegarden
+
+### Usage
+ * Make sure to have `docker`, `docker-compose`, and `yarn` installed locally.
+ * After a clone, run `./scripts/update` to populate the database and install node modules.
+ * Run `./scripts/server` to run the development server, which will be accessible on port **9001**, not 3000 as listed by the output from claudia-local-api. Several example endpoints have been defined in `src/api.js`.
+ * To publish, make sure you have specified valid AWS credentials and have listed the proper database credentials in a `.env` file in the root of the project. The format for this file can be copied from `env-template`. You can then run `./scripts/publish` to publish to AWS Lambda.
```

Comment:
I see you have the `.aws` directory mapped in the `tiler` config in `docker-compose.yml`, but I'm not clear on what credentials it's actually looking for.  Does Claudia have a default profile name that it uses, or is it looking for `AWS_PROFILE` in the environment?  If the latter, that should probably be added to the `.env` template.

Reply:
Claudia uses whichever AWS profile you have labeled "default", I'll make that clear in the readme and set up specifying a profile from .env.